cms/templates/cms/home_page.html

```diff
@@ -44,97 +45,134 @@
 
       <div class="banner-wrapper">
         <div class="banner-wrapper-content">
-          <h1>
-            <div class="banner-image">
-              <img src="{% static 'images/logo-mit-verylg@2x.png' %}" alt="MIT" />
-            </div>
+          <h3>
             {{ title }}
-          </h1>
+          </h3>
         </div>
       </div>
 
       <div class="row sub-banner">
-        <div class="col-md-12">
-          <div class="sub-banner-content">
-            MIT has created a new credential for online learners: the MicroMasters, which will be granted by <a href="https://www.edx.org/school/mitx" target="_blank">MITx</a> to students who do exceptionally well in a given set of graduate-level online courses and do well in a subsequent exam. <a href="javascript:void()" data-target="#modal-about" data-toggle="modal">Learn more</a>.
+        <div class="col-md-12 ">
+          <div class="sub-banner-section">
+            <h4>What is the MicroMasters program?</h4>
+            <div class="sub-banner-text">
+              <p>The MicroMasters credential is a new option for online learning in which the students can earn
+                a full master's degree by taking about half the course content online and half on campus.</p>
+              <p>The MicroMasters Credential will be granted by MITx (MIT's online learning initiative)
+                to students who do exceptionally well in a set of graduate-level online courses and a subsequent exam.
+                For many learners, the MicroMasters credential can be a step toward earning an MIT master's degree at a
+                great cost saving.</p>
+            </div>
+            <div class="sub-banner-logos">
+              <div class="sub-banner-logos-text">Powered by</div>
+              <img width="100px" src="{% static 'images/edx_logo.png' %}">
+              <img width="150px" src="{% static 'images/micromasters_logo.png' %}">
+
+            </div>
           </div>
         </div>
       </div>
+      <div class="row reasons-section">
+
+        <h4>Why should I enroll?</h4>
+        <br>
+        <div class="col-md-3 col-md-offset-1">
+          <div class="reasons-images-wrap">
+             <img height="100px" src="{% static 'images/online.png' %}">
+          </div>
+
+          <h5>ONLINE LEARNING</h5>
+          <p>Take graduate level courses remotely, from anywhere, and meet other students from around the world.</p>
```

Comment:
In the mockups `graduate level` is `graduate-level` so we should add a hyphen here too